src/components/content/common/catalog/userServiceProps.ts

```diff
@@ -0,0 +1,43 @@
+import { UserOrderableServiceVo } from '../../../../xpanse-api/generated';
```

Comment:
this class can be moved under order/services/ package

Reply:
Ok.